pkgs/development/libraries/science/math/nccl/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, pkgs, fetchurl, cudatoolkit }:
+
+with pkgs;
+
+stdenv.mkDerivation rec {
+  name = "cudatoolkit-${cudatoolkit.majorVersion}-nccl-${version}";
+  version = "1.3.4-1";
+
+  src = fetchurl {
```

Comment:
`fetchFromGitHub` should be used instead https://nixos.org/nixpkgs/manual/#sec-sources